script/Ciel/Day 9/09_01_CIEL09_3_2.txt

```diff
@@ -296,120 +350,137 @@ You understand what I'm saying, right, Shiki-kun?
 [sha:a7260bec529a753d9049960f361aaee47b7f6333]{
 -- Page 365, Offset 23080.
 -- 吸血鬼になったモノは速やかに死ぬべきなの。
-People that turn into vampires should just die immediately.
+Anyone that turns into a vampire should just fucking die already.
 }
 [sha:f5f9290a2affe2f30f448c8381617db8a7843f5c]{
 -- Page 365, Offset 23081.
 -- コイツらはゴミ以下なの。病原菌なの。ただそこにいるだけで汚物を垂れ流す害悪なのよ……！」
-They're worse than trash. They're germs. They're a blight that oozes filth just by existing...!"
+They're worse than trash. They're germs.
+ They're a blight whose mere existence sullies the world around them...!"
 }
 [sha:7e0ea34b7a2828a4516739f12eb472cd3b27e12d]{
 -- Page 366, Offset 23082.
 -- 部屋の隅にあった鉄槌が持ち上げられる。
-She lifts an iron hammer from the corner of the room.
+She picks up the iron hammer that was lying in the corner of the room.
```

Comment:
I think this is referring to the halberd, as bizarre as that may sound. 鉄槌 used in its metaphorical form. Though I will admit, it's a pretty strange way of writing things.

Reply:
Yeah, it definitely is, but I'm not about to rewrite the metaphor. I think it's pretty clear.

Reply:
Is it? Was confusing to me before I looked at the Japanese.
鉄槌 makes me think of like, judgment, and judicial stuff:
>She picks up the instrument of justice that had been lying in the corner of the room.

Reply:
Did you play it in game? Pretty sure it's pictured

Reply:
![iron hammer ingame](https://user-images.githubusercontent.com/98728981/200096585-fe5aa550-d024-43f2-844d-402141808558.png)

Reply:
It's clarified on the same page and pictured immediately after. I don't think there's a need to editorialize. 
![2022-11-04-221402_1920x1140_scrot](https://user-images.githubusercontent.com/2476799/200096647-fd3cb33e-cd5a-4a9a-92cd-27ae1bbbc187.png)
![2022-11-04-221418_1920x1140_scrot](https://user-images.githubusercontent.com/2476799/200096657-743b3279-5628-49b1-b0e0-7468d00baf99.png)

Reply:
It's just something which was far more clear to me in JP compared to EN, so it felt like a weird way of phrasing things. Especially since it's what introduces the halberd into the scene. An iron hammer doesn't exactly call to mind a halberd, is all.

Reply:
Okay, I take it back. It does feel weird to refer to it bluntly two different ways that quickly.